test/e2e/autoscaling/cluster_size_autoscaling.go

```diff
@@ -881,6 +881,19 @@ var _ = SIGDescribe("Cluster size autoscaling [Slow]", func() {
 			clusterSize = manuallyIncreaseClusterSize(f, originalSizes)
 		}
 
+		// If new nodes are disconnected too soon, they'll be considered not started
+		// instead of unready, and cluster won't be considered unhealthy.
```

Comment:
healthy

Reply:
We want the cluster to become unhealthy in this scenario, to test back-off from adding more nodes